docs/DeveloperGuide.md

```diff
@@ -287,6 +312,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Add a tag**
+
+**MSS**
+
+1.  User requests to create a tag
+2.  Projact to check if the tag exists and creates the tag if it has not been created.
+
+    Use case ends.
+    
+**Extensions**
+
+* 2a. The tag has been created.
+
+    * 2a1. Projact shows an error message.
```

Comment:
Why will there be an error message after the tag is created (Line 326-328)? Maybe you can write it this way(?):
MSS
1. User requests to add a tag.
2. Projact adds a tag.
Use case ends.

Extensions
2a. The tag already exists.
2a1. Projact shows an error message.
Use case ends.

Is this clearer haha and also must we show in the use case that there are 2 ways to add a tag (like in user guide) or a general one (seen above) is ok?

Reply:
Ok! I can change to what was suggested. For the showing of 2 ways, I think a general one will be fine? Since like the use case is to show interaction between system and user.